inference-engine/tests/unit/vpu/middleend_tests/passes_tests/adjust_data_batch_tests.cpp

```diff
@@ -0,0 +1,389 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vpu/middleend/allocator/allocator.hpp>
+#include <vpu/stages/mx_stage.hpp>
+#include <vpu/utils/numeric.hpp>
+
+#include "graph_transformer_tests.hpp"
+
+using namespace vpu;
+
+class VPU_AdjustDataBatchTest_New_Inf : public GraphTransformerTest {
```

Comment:
`New_Inf` suffix - what does it mean?

Reply:
New_Inf ~ new tests infrastructure

Reply:
But it is clear that it is VPU and it is new infrastructure (it is deduced from the path of the test file)

Reply:
Okay, I'll remove the suffix.